tests/functional/modules/ims_zddl/test_ims_zddl_invalid_input.py

```diff
@@ -0,0 +1,266 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function)
+from pprint import pprint
+import pytest
+from ibm_zos_ims.tests.functional.module_utils.ims_test_zddl_utils import ZDDLInputParameters as ip
+
+__metaclass__ = type
+
+"""
+Work flow for Combination functional tests goes as follows:
+------------------------------------ Running on BMP ------------------------------------
+1. Invalid IMS ID                                   Online
+2. Invalid RESLIB -                                 Online
+3. Invalid PROCLIB -                                Online
+4. Invalid STEPLIB -                                Online
+5. Invalid SQL Input                                Online
+6. Empty SQL Input(EMPTY LIST)                      Online
+7. Valid and invalid SQL Input 
+8. No proclib
+9. No Sql_Input (SQL INPUT AS NONE)
+10. No commit
+11. Program view with no database
+------------------------------------ Not supporting DLI, so these are commented ------------------------------------
+12. Invalid RESLIB                                  Offline
+13. Invalid PROCLIB                                 Offline
+14. Invalid STEPLIB                                 Offline
+15. Empty STEPLIB                                   Offline
+16. Invalid SQL Input                               Offline
+17. Empty SQL Input                                 Offline
+18. Valid IMS ID but offline
+19. Invalid IRLM ID                                  
+"""
+
+# ------------- VARIABLES
+# "DEFAULT" VALUES
+ONLINE = ip.ONLINE 
+OFFLINE = ip.OFFLINE            
+IMS_ID = ip.IMS_ID              
+IRLM_ID = ip.IRLM_ID          
+RESLIB = ip.RESLIB           
+PROCLIB = ip.PROCLIB 
+STEPLIB = ip.STEPLIB
+
+# ------------- SQL Variables
+SQL_FUll1 = "IMSTESTL.SEQ.SQLFRS1" 
+SQL_SIMPLE = "IMSTESTL.SEQ.SQLSIN" 
+SQL_SPORT = "IMSTESTL.SEQ.SQLSPRT"  
+SQL_DELETE = "IMSTESTL.SEQ.SQLDLT"  
+SQL_EMPTY = "IMSTESTL.SEQ.SQLMPT"
+SQL_INVALID = "IMSTESTL.SEQ.SQLINVLD"
+SQL_MIXED = "IMSTESTL.SEQ.SQLMXVL"
+SQL_NO_COMMIT = "IMSTESTL.SEQ.SQLNCM" 
+SQL_NO_DB_PRGVIEW = "IMSTESTL.SEQ.SQLNCS" 
+
+# Control statements
+VERBOSE = ip.VERBOSE
+AUTO_COMMIT = ip.AUTO_COMMIT
+SIMULATE = ip.SIMULATE
+dynamic_programview = ip.dynamic_programview
```

Comment:
We updated the parameter name in class `ZDDLInputParameters` - however when it's referenced here, it's still the old lowercase name. Does that work? Also shouldn't control variable name be all upper case for consistency with other variables on lines 56-58?

`DYNAMIC_PROGRAMVIEW = ip.DYNAMIC_PROGRAMVIEW`